botfront/imports/api/graphql/story/mongo/stories.js

```diff
@@ -42,3 +42,40 @@ export const searchStories = async (projectId, language, search) => {
     ).fetch();
     return matched;
 };
+
+const replaceLine = (story, lineToReplace, newLine) => {
+    // regexp: [ ] = space; + = any number of the characters in the []; $ = end of string
+    const regex = new RegExp(`- ${lineToReplace}([ ]+\n|\n|[ ]+$|$)`, 'g');
```

Comment:
Minor: I think instead of `[ ]+\n|\n|[ ]+$|$`  a shorted and simpler regex would be   `\s*\n?$`, that translate to:  match 0 or more spaces then 0 or 1 \n until the end of the string, but maybe I misunderstood the original one ?